sql/firefox_accounts/fxa_amplitude_email_clicks/view.sql

```diff
@@ -0,0 +1,220 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.firefox_accounts.fxa_amplitude_email_clicks`
+AS
+WITH hmac_key AS (
+  SELECT
+    AEAD.DECRYPT_BYTES(
+      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
+      ciphertext,
+      CAST(key_id AS BYTES)
+    ) AS value
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`
+  WHERE
+    key_id = 'fxa_hmac_prod'
+),
+sendjobs_numbered AS (
+  SELECT
+    *,
+    ROW_NUMBER() OVER (PARTITION BY SendID ORDER BY snapshot_date DESC) AS _n
```

Comment:
Are there docs or context for these tables? This looks to me like this query would use all of history every time, but I'm not sure how bad that is here.

Reply:
Access is fairly locked down to these tables and to the documentation around them (there's a google doc). The tables are quite small, and yes this is windowing over all history in the table. I will add some comments to explain that context.